.github/workflows/publish.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish
+
+on:
+  push:
+    branches:
+      - 12/publish-step
+
+      # - master
+
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2 # https://github.com/actions/checkout
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v1 # https://github.com/actions/setup-node
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org
+
+      - name: Install
+        run: yarn --offline
+
+      - name: Build
+        run: yarn build
+
+      - name: Continuous integration check
+        run: yarn ci-check
+
+      # Dry run - `yarn lerna version --conventional-commits --no-changelog --no-git-tag-version --no-push --yes`
+      - name: Publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
We need to get the correct npm access token and add it to https://github.com/carbon-design-system/ibm-cloud-cognitive/settings/secrets before merging